datafusion/expr/src/built_in_function.rs

```diff
@@ -429,31 +341,130 @@ impl BuiltinScalarFunction {
     }
 }
 
-impl fmt::Display for BuiltinScalarFunction {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        for (func_name, func) in NAME_TO_FUNCTION.iter() {
-            if func == self {
-                return write!(f, "{}", func_name);
-            }
+fn aliases(func: &BuiltinScalarFunction) -> &'static [&'static str] {
```

Comment:
Sorry I commented on the wrong line previously @comphead , should be here.

Adding a comment for aliases():
`/// First alias in the array is used to display function names`